bsms.mediawiki

```diff
@@ -280,74 +280,104 @@ wsh(sortedmulti(2,[4b6a9b93/48'/1'/1']tpubDDpNnt2fg3G4ukE8gsifBKank5sGemyMDa6gUj
 ===Mode: <tt>EXTENDED</tt> Encryption===
 ====ROUND 1====
 *Coordinator
-** M-of-N: 1/2
+** M-of-N: 2/3
 ** ADDRESS_TYPE: NESTED_SEGWIT
-** TOKEN for Signer 1 (hex): 2d5d8de454dede3e266c295b
-*** TOKEN (mnemonic): coil unaware jungle predict unknown business often lunch forest
-*** TOKEN (decimal): 14039925608831991734145395035
-*** ENCRYPTION_KEY (hex): a3b3e3961f035fce44cd38454b6e4db42ffb64f80287beaca0f95202929b4867
-** TOKEN for Signer 2 (hex): 3b7af42cd4cc259d1035557b
-*** TOKEN (mnemonic): derive stumble arctic praise seat soldier dolphin primary knee
-*** TOKEN (decimal): 18408257613864575438530368891
-*** ENCRYPTION_KEY (hex): 8eb61f8aff1b919c1a04b7826bd4829fdba0a28c1614fd4b84870f3d0051efc0
+** TOKEN for Signer 1 (hex): 27bfc197f7a3d979201adfdb
+*** TOKEN (decimal): 12301734434612746087128817627
+*** TOKEN (mnemonic): chicken wrap grass urban diesel route lesson fossil swap
+*** ENCRYPTION_KEY (hex): afb2877099f1faf16fbbc3e9f3acc769c3c52511e172ebebc9ff6da731024b95
+** TOKEN for Signer 2 (hex): 318bb031160659658d05bf06
+*** TOKEN (decimal): 15333638212150840703511281414
+*** TOKEN (mnemonic): cover frown arrange clown grain ready cross retire almost
+*** ENCRYPTION_KEY (hex): 7e0470b6d747367492d4b5b1b2df62285ec90bb4d5753382939f00f499038139
+** TOKEN for Signer 3 (hex): 88c3db1a6897513ee5bba1c7
+*** TOKEN (decimal): 42326736555939458236399985095
+*** TOKEN (mnemonic): match burst shoot spell insect panda notice injury shuffle
+*** ENCRYPTION_KEY (hex): 62bd9dff25b07f32483703b2a12932e158dcaf14406eb19d87adef281dd19bf0
 
 * Signer 1
-** MASTER_KEY_FINGERPRINT: 112528b3
-** PRIVATE_KEY (m/48’/1’/1’): L2x249XLMPvhTN6gtXzeTahPYyNENrBxETxehfA5PgqYDCJoYViU
-** XPUB (m/48’/1’/1’): xpub6BoNx2A7W1JfqLzrqWMMFrVfhCdthUJdheXAk3UuUhUsPKs11ZK1RgjeqdQVed12tCJzUrf3G1gQtR9zTc9fN8iM8Trvd7FXQ5pg23aiCfs
+** MASTER_KEY_FINGERPRINT: d91b75fb
+** PRIVATE_KEY (m/48’/1’/1’): L4MHufXgyCgucvFYQeqQqmso2yrqBYxQC9SFR4fegqGLUtMt4vwZ
+** XPUB (m/48’/1’/1’): xpub6C9STF1ptpgycxov97hKbR45fGiPm6vrYjTLvHSAzHpHpDW1afckbSQKACCmpReMtWpbEVdhGHT5btD9oXaaRWXs4fQ8SVRy8RFNh53kX7g
 ** Legacy signature
 ** <tt>signer_1_key.bsms</tt>:
 <pre>BSMS 1.0
-2d5d8de454dede3e266c295b
-[112528b3/48'/1'/1']xpub6BoNx2A7W1JfqLzrqWMMFrVfhCdthUJdheXAk3UuUhUsPKs11ZK1RgjeqdQVed12tCJzUrf3G1gQtR9zTc9fN8iM8Trvd7FXQ5pg23aiCfs
-Signer 1 key
-IIr4zrfx2gbDPf4IWmKWRYO7dt19QyVDFGcHCdlf3LlZawztdOXJ6abPivYl8eV0uWsIwzBCF6af9urwtUYZjI8=</pre>
+27bfc197f7a3d979201adfdb
+[d91b75fb/48'/1'/1']xpub6C9STF1ptpgycxov97hKbR45fGiPm6vrYjTLvHSAzHpHpDW1afckbSQKACCmpReMtWpbEVdhGHT5btD9oXaaRWXs4fQ8SVRy8RFNh53kX7g
+Signer 0 key
```

Comment:
Signer 1 key

Reply:
fixed in https://github.com/nunchuk-io/bips/commit/8ddce677ea974632007b6fc549ee5ed5d63e5a79